### PR TITLE
Add Gitcoin security email address

### PR DESCRIPTION
**Pull request by captnseagraves**

### CODE REVIEW

**Comment by corpetty** [comment]
@captnseagraves you're missing a `|` in that commit.

File: Dapps.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 | Democracy Earth | | hello@democracy.earth | |
 | Dharma | | security@dharma.io | |
 | Ethfinex | | bounty@ethfinex.com | |
+| Gitcoin | security@gitcoin.co |
```
Comment (corpetty) [inline Dapps.md]: should have an extra `|` before your email.

**Review by corpetty — DISMISSED** [review]
need extra pipe

**Comment by owocki** [comment]
@captnseagraves is it worth adding a link to our bug bounty ? ref: https://github.com/gitcoinco/web/issues/38

**Review by fubuloubu — APPROVED** [review]

**Comment by corpetty** [comment]
@captnseagraves did you want to add the gitcoin bug bounty?  otherwise I'll just merge it.

**Pull request merged**